classes/class-dintero-checkout-gateway.php

```diff
@@ -92,7 +92,7 @@ public function init_form_fields() {
 		 * @return string
 		 */
 		public function get_icon() {
-			return '<img src="' . esc_attr( dintero_get_brand_image_url() ) . '" style="max-width: 90%" alt="Dintero Logo" />';
+			return '<a href="https://www.dintero.com" target="_blank" title="' . dintero_keyword_backlinks() . '"><img src="' . esc_attr( dintero_get_brand_image_url() ) . '" style="max-width: 90%" alt="Dintero Logo" /></a>';
```

Comment:
Do we also want to add some styling here to override any global styles on the `<a>`-tag. Eg. removing margins, padding, border, underlines, shadows, outlines. Also when the element has `:hover`,`:focus` or is  `:active` ?